source/rmarkdown/metadata_bodemstalen_fysicochemisch.Rmd

```diff
@@ -92,75 +172,18 @@ high_cor_pairs <- data.frame(
 )
 high_cor_pairs <- high_cor_pairs[!duplicated(t(apply(high_cor_pairs[,1:2], 1, sort))),]
 high_cor_pairs <- high_cor_pairs[order(-abs(high_cor_pairs$Correlation)),]
-print(high_cor_pairs)
-```
-
-
-```{r summary-stats}
-summary_stats <- df_numeric_clean %>%
-  summarise(across(everything(), list(
-    mean = ~mean(., na.rm = TRUE),
-    sd = ~sd(., na.rm = TRUE),
-    min = ~min(., na.rm = TRUE),
-    max = ~max(., na.rm = TRUE)
-  )))
-print(summary_stats)
-
-```
-
-
-Calculate skewness and kurtosis
-
-```{r}
-skewness_values <- apply(df_numeric, 2, function(x) ifelse(all(is.na(x)), NA, moments::skewness(x, na.rm = TRUE)))
-kurtosis_values <- apply(df_numeric, 2, function(x) ifelse(all(is.na(x)), NA, moments::kurtosis(x, na.rm = TRUE)))
-
-skew_kurt <- data.frame(
-  variable = names(df_numeric),
-  skewness = skewness_values,
-  kurtosis = kurtosis_values
-)
-```
-
-
-Print skewness and kurtosis
-
-```{r}
-print(skew_kurt)
-
+kable(high_cor_pairs, digits = 2)
 ```
 
+Variance inflation factors:
 
-Create histograms and Q-Q plots for each numeric variable
-
-```{r}
-for (col in names(df_numeric)) {
-  png(paste0(col, "_diagnostic_plots.png"), width = 800, height = 400)
-  par(mfrow = c(1, 2))
-  
-  # Histogram
-  hist(df_numeric[[col]], main = paste("Histogram of", col), xlab = col)
-  
-  # Q-Q plot
-  qqnorm(df_numeric[[col]], main = paste("Q-Q Plot of", col))
-  qqline(df_numeric[[col]], col = "red")
-  
-  dev.off()
-}
-
+```{r multicollinearity}
```

Comment:
obv de significante paarsgewijze correlaties tussen de C en N variabelen, veronderstel ik dat we bijvoorbeeld TN, Ndensity, TOC, en Nstock kunnen verwijderen, en Cdensity en C_N_stockbased kunnen behouden? Of dus een selectie moeten maken. De vif voor Cdensity is momenteel zeer hoog, maar ik vermoed dat die lager zal zijn als we bovenstaande variabelen verwijderen en de multicollineariteit opnieuw testen zonder deze variabelen?

Reply:
Ja, inderdaad. Het is een iteratief proces. Je verwijdert sterk gecorreleerde variabelen of variabelen met hoge VIF en test opnieuw.